### PR TITLE
Batch size

### PR DESCRIPTION
**Pull request by paulocilasjr**
## Summary

Simplifies the display of auto-selected `batch_size` and `learning_rate` values in the configuration report table.

## Changes

- Displays the resolved auto-selected value on its own line.
- Shows `Auto-selected by Ludwig` when no selection is made by the user.
- Adds tests for the updated batch size and learning rate output.

### CODE REVIEW

File: tools/imagelearner/test_image_workflow.py [hunk 1]
```diff
@@ -153,3 +153,39 @@ def test_metric_summary_formats_hits_at_3_without_changing_metric_values():
...
+
+    assert (
+        "<span style='font-size: 0.85em;'>"
+        "Auto-selected batch size by Ludwig:</span>"
+    ) in html
+    assert ">16</td>" in html
+    assert "<span style='font-size: 0.85em;'>16</span>" not in html
+    assert "Based on model architecture and training setup" not in html
+    assert "Ludwig Trainer Parameters" not in html
+    assert ">auto</td>" not in html
```
Comment (qchiujunhao) [inline tools/imagelearner/test_image_workflow.py:174]: This assertion fails on the PR head. With `training_progress` present and no `learning_rate` in the test config, `format_config_table_html()` also renders the Learning Rate row as `<br>auto</td>`, so `pytest tools/imagelearner/test_image_workflow.py::test_config_table_displays_resolved_auto_batch_size` fails. Scope this check to the Batch Size row, or include a learning-rate value in the test input.

**Review by qchiujunhao — CHANGES_REQUESTED** [review]
Found one blocker: the PR head currently fails its new focused Image Learner test. The assertion needs to be scoped to the batch-size row, or the test input should provide a learning-rate value so the unrelated auto learning-rate row is not emitted.

File: tools/imagelearner/test_image_workflow.py (hunk 1 above)
Comment (paulocilasjr) [reply tools/imagelearner/test_image_workflow.py:174]: Followed the suggestion: batch-size test now provides an explicit value for learning_rate.

**Review by qchiujunhao — APPROVED** [review]
The previously flagged test issue is fixed, the focused Image Learner test passed locally, and the current GitHub checks are green.
